viewproduct.php

```diff
@@ -10,162 +10,157 @@
+    $result_select_random_products = mysqli_stmt_get_result($stmt_related);
+}
             ?>
          <!-----------------Side Bar----------------->
-         <div class="sidebar"  style=" border-style:solid; border-width:1px; border-color:
-         #099;">
-            <div class="col_1">
-               <h2>Similar Products</h2>
-               <div class="box" style="background-color:#FFF;">
-                   <?php
-                      //Count number of rows
-                      	$no_of_select_random_products = mysqli_num_rows($result_select_random_products);
-                      	if($no_of_select_random_products>0)
-                      	{
-                       	///////////Fetch products from variavle and display
-                      	while($data = mysqli_fetch_array($result_select_random_products))
-                      	{
-                      ?>
-                      	<p>
-     <div class="row"> 
-                         <div class="col_3">
+<div class="sidebar" style="border: 1px solid #099;">
+    <div class="col_1">
+        <h2>Similar Products</h2>
+        <div class="box" style="background-color:#FFF;">
+            <?php
+                // Count number of rows
+                $no_of_select_random_products = mysqli_num_rows($result_select_random_products);
+                if ($no_of_select_random_products > 0) {
+                    // Fetch products from variable and display
+                    while ($data = mysqli_fetch_array($result_select_random_products)) {
+            ?>
+                <p>
+                    <div class="row"> 
+                        <div class="col_3">
                             <img src="<?php echo $data['ProductImage']; ?>" width="80" height="80">
-                         </div>
-                         <div class="col_3" style="color:#000;">
-                            <?php echo  $data['ProductName']; ?>
-                            <br>
+                        </div>
+                        <div class="col_3" style="color:#000;">
+                            <?php echo $data['ProductName']; ?><br>
                             LKR:<?php echo $data['ProductPrice']; ?>
-                         </div>
-                         </div>
-                   </p>
-                         <a href="viewproduct.php?pid=<?php echo $data['ProductID']; ?>"><input type="button" value="See  Details"></a>
-                      </form> 
-                   <?php } } else {?>
-                   <h3>Oops! No Products Found!</h3>
-                   <?php } ?>	
-                </div>
-               </div>
+                        </div>
+                    </div>
+                </p>
+                <a href="viewproduct.php?pid=<?php echo $data['ProductID']; ?>"><input type="button" value="See Details"></a>
+            <?php } } else { ?>
+                <h3>Oops! No Products Found!</h3>
+            <?php } ?>    
         </div>
+    </div>
+</div>
+
+
```

Comment:
- Move the inline styles to a separate CSS file for better maintainability.
- Remove the unnecessary <form>
- tag since there is no form element in the code.
- Use prepared statements instead of directly passing variables to SQL queries to prevent SQL injection attacks.